### PR TITLE
 [11.0][MIG] stock_picking_sale_order_link

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Review by tafaRU — APPROVED** [review]
Code review and test.

File: stock_picking_sale_order_link/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models
```
Comment (rousseldenis) [inline stock_picking_sale_order_link/models/stock_picking.py]: odoo

File: stock_picking_sale_order_link/tests/test_stock_picking_sale_order_link.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
```
Comment (rousseldenis) [inline stock_picking_sale_order_link/tests/test_stock_picking_sale_order_link.py]: odoo

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comment

**Comment by sergio-teruel** [comment]
@rousseldenis changes  done, edit it from  my  Mobile... :smile:

**Review by pedrobaeza — APPROVED** [review]
Merging as @rousseldenis's comments are attended

**Pull request merged**